rhombus/scribblings/ref-syntax-class.scrbl

```diff
@@ -36,17 +36,23 @@
  accessed from @rhombus[id, ~var] using
  @list[@rhombus[id, ~var], @rhombus[.], @rhombus[attr_id, ~var]]. To use an attribute
  within a template, parentheses are needed around the variable name,
- @rhombus[.], and attribue name to group them together if the variable
+ @rhombus[.], and attribute name to group them together if the variable
  name is preceded by a @rhombus[$] escape:
  @rhombus[$($$(@list[@rhombus[id, ~var], @rhombus[.], @rhombus[attr_id, ~var]]))].
 
  A variable bound with a syntax class (within a syntax pattern) can be
  used without dot notation. In that case, the result is a sequence of
  syntax objects corresponding to the entire match of a
- @rhombus[syntax_pattern], as opposed to an indvidual attributes within
+ @rhombus[syntax_pattern], as opposed to an individual attributes within
  the match. Use @rhombus[...] after a @rhombus[$]-escaped reference to
  the variable in a syntax template.
 
+Custom attributes of a syntax class can be defined in a block following a 
```

Comment:
Besides this text, the `doc` form that defines the `syntax.class` grammar should be updated. Probably you want to replace `syntax_pattern` with `clause`, where clause is defined to be either `syntax_pattern` or `syntax_pattern: pattern_body; ...` and `pattern_body` is an expression or definition of `~attr` form. See `def` for an example of how to define helper nonterminals like `clause`.